src/resolver/resolve_path.zig

```diff
@@ -1728,72 +1724,6 @@ test "longestCommonPath" {
     _ = t.expect("/app/public/", longestCommonPath(more[0..2]), @src());
 }
 
-pub fn basename(path: []const u8) []const u8 {
-    if (path.len == 0)
-        return &[_]u8{};
-
-    var end_index: usize = path.len - 1;
-    while (isSepAny(path[end_index])) {
-        if (end_index == 0)
-            return &[_]u8{};
-        end_index -= 1;
-    }
-    var start_index: usize = end_index;
-    end_index += 1;
-    while (!isSepAny(path[start_index])) {
-        if (start_index == 0)
-            return path[0..end_index];
-        start_index -= 1;
-    }
-
-    return path[start_index + 1 .. end_index];
-}
-pub fn lastIndexOfSep(path: []const u8) ?usize {
-    if (comptime !bun.Environment.isWindows) {
-        return strings.lastIndexOfChar(path, '/');
-    }
-
-    return std.mem.lastIndexOfAny(u8, path, "/\\");
-}
-
-pub fn nextDirname(path_: []const u8) ?[]const u8 {
-    var path = path_;
-    var root_prefix: []const u8 = "";
-    if (path.len > 3) {
-        // disk designator
-        if (path[1] == ':' and isSepAny(path[2])) {
-            root_prefix = path[0..3];
-        }
-
-        // TODO: unc path
-
-    }
-
-    if (path.len == 0)
-        return if (root_prefix.len > 0) root_prefix else null;
-
-    var end_index: usize = path.len - 1;
-    while (isSepAny(path[end_index])) {
-        if (end_index == 0)
-            return if (root_prefix.len > 0) root_prefix else null;
-        end_index -= 1;
-    }
-
-    while (!isSepAny(path[end_index])) {
-        if (end_index == 0)
-            return if (root_prefix.len > 0) root_prefix else null;
-        end_index -= 1;
-    }
-
-    if (end_index == 0 and isSepAny(path[0]))
-        return path[0..1];
-
-    if (end_index == 0)
-        return if (root_prefix.len > 0) root_prefix else null;
-
-    return path[0 .. end_index + 1];
-}
-
```

Comment:
☝️ Removed `nextDirname` from `resolve_path` in favor of `std.fs.path.dirname` but need a second set of 👀  on this 🙏